src/t_hash.c

```diff
@@ -551,18 +544,21 @@ robj *hashTypeDup(robj *o) {
 
 /* callback for to check the ziplist doesn't have duplicate records */
 static int _hashZiplistEntryValidation(unsigned char *p, void *userdata) {
```

Comment:
This function now needs to be renamed too (and comment updated).
it's not only a validation helper function, but also a conversion helper function.

Reply:
Fixed.